_scripts/migrations/0015_discussion_access_and_invites.sql

```diff
@@ -0,0 +1,98 @@
+ALTER TABLE discussions ADD COLUMN IF NOT EXISTS public_access boolean default true not null;
+
```

Comment:
creating tables that hold flair gating, invites, requests for access, user_level access, etc.